agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptionsParser.java

```diff
@@ -156,6 +156,9 @@ private void handleOption(AgentOptions options,
 						value)) {
 			return;
 		}
+		if (key.startsWith("proxy-") && TeamscaleProxyOptions.handleTeamscaleProxyOptions(options.teamscaleProxyOptions, key, value, filePatternResolver)) {
```

Comment:
IMO its not necessary to resolve the finding regarding the length of this function. We don't tolerate findings regarding method length, so I would just ignore it.